packages/@storefront/flux-capacitor/test/unit/core/adapters/search.ts

```diff
@@ -230,35 +231,63 @@ suite('Search Adapter', ({ expect, stub }) => {
       const max = stub(ConfigAdapter, 'extractMaxRefinements').returns(2);
       const config = stub(Selectors, 'config').returns(conf);
       const navigations: any = [
-        { name: 'Adds non-selected refinement', refinements: [4], selected: [], more: false },
-        { name: 'Adds selected refinements', refinements: [6, 7, 4, 5, 6, 8], selected: [1, 5] },
-        { name: 'Adds selected refinement first', refinements: [8, 9], selected: [1], more: true },
-        { name: 'Limits to max', refinements: [0, 1, 2, 3, 4], selected: [1, 2, 3, 4], more: true },
-        { name: 'Limits to max', refinements: [0, 1, 2, 3, 4], selected: [], more: true },
+        { field: 'Adds non-selected refinement', refinements: [4], selected: [], more: false },
+        { field: 'Adds selected refinements', refinements: [6, 7, 4, 5, 6, 8], selected: [1, 5] },
+        { field: 'Adds selected refinement first', refinements: [8, 9], selected: [1], more: true },
+        { field: 'Limits to max', refinements: [0, 1, 2, 3, 4], selected: [1, 2, 3, 4], more: true },
+        { field: 'Limits to max', refinements: [0, 1, 2, 3, 4], selected: [], more: true },
       ];
+      stub(ConfigAdapter, 'extractToggleNavigations').returns([]);
 
+      // tslint:disable
```

Comment:
Only disable the rule that you need (probably `max-line-length` in this case).

Reply:
I wrote it this way so it would disable for multiple lines, to do what you want we need to disable max-line-length on each line

Reply:
`// tslint:disable max-line-length` seems to work to disable multiple lines.